packages/fxa-content-server/server/lib/sentry.js

```diff
@@ -89,16 +89,29 @@ if (config.get('sentry.dsn')) {
  */
 function tryCaptureValidationError(err) {
   try {
-    const errorDetails =
-      err?.details instanceof Map && err?.details?.get('body');
+    // Try to get error details. This might not be present.
+    let errorDetails = null;
+    if (err != null && err.details instanceof Map) {
+      errorDetails = err.details.get('body');
+    }
 
     if (errorDetails) {
       const message = `${errorDetails}`;
```

Comment:
~Q: Is this used somewhere that I'm not seein'?~
Ah, seems to be used on L119 below and is some stringified version of the `errorDetails` object.